examples/forms.amp.html

```diff
@@ -238,15 +238,12 @@
     </fieldset>
 
     <div submit-success>
-        <template type="amp-mustache">
-            Success! Thanks {{name}} for subscribing! Please make sure to check your email {{email}}
-        to confirm!
-        </template>
+      Success! Thanks for subscribing! Please make sure to check your email
```

Comment:
let's keep one use a template and one without. 

maybe `success` can stay as it was and `error` can be done without a template showing some text and a "error icon" with amp-img.